jobs/build/advisories/Jenkinsfile

```diff
@@ -116,7 +116,9 @@ node {
                 lib.create_advisory("rpm")
                 if (params.VERSION.startsWith("4")) {
                     lib.create_advisory("extras")
-                    lib.create_advisory("metadata")
+                    if (params.VERSION != "4.6") {
```

Comment:
We'll run into this with 4.7 again.

Reply:
Right, better to use a condition like `>= 4.6`. See `commonlib.extractMajorMinorVersionNumbers` to convert the version string to numbers.